quartz/plugins/transformers/ofm.ts

```diff
@@ -176,13 +176,17 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
           const anchor = rawHeader?.trim().replace(/^#+/, "")
           const blockRef = Boolean(anchor?.startsWith("^")) ? "^" : ""
           const displayAnchor = anchor ? `#${blockRef}${slugAnchor(anchor)}` : ""
-          const displayAlias = rawAlias ?? rawHeader?.replace("#", "|") ?? ""
+          let displayAlias = rawAlias ?? rawHeader?.replace("#", "|") ?? ""
           const embedDisplay = value.startsWith("!") ? "!" : ""
 
           if (rawFp?.match(externalLinkRegex)) {
             return `${embedDisplay}[${displayAlias.replace(/^\|/, "")}](${rawFp})`
           }
 
+          if (displayAlias && displayAlias.startsWith("|")) {
```

Comment:
can you add a lil note here explain this case?

Reply:
What is  lil note ? 

1. this `if` block to handle this case:  `wikilink no alias,  using block id as alias` 
![image](https://github.com/jackyzha0/quartz/assets/11496772/3d2a5dac-67dd-4e67-877e-1bc2cb1baec9)

for this situation
```markdown
| plain note                        | display        |
| --------------------------------- | -------------- |
| [[transclude_1#^2emjcu]]          | no alias       |
| ![[transclude_1#^2emjcu]]         | no alias embed |
``` 
I don't set an alias for this ref,  when code go to `line 186`,  the `displayAlias === |^2emjcu` (seems auto generate ?),  the final result `[[transclude_1#^2emjcu|^2emjcu]]`  . It will render error ,  so I add `line 186` if block to handle this case.

Reply:
> can you add a lil note here explain this case?

if remove `line 186`, will be rendered like 
![image](https://github.com/jackyzha0/quartz/assets/11496772/18b5ce67-fa56-45ad-9883-b79415694868)

Reply:
I mean a comment :)